pages/common/msmtp.md

```diff
@@ -0,0 +1,16 @@
+# msmtp
+
+> An SMTP client.
+> In the default sendmail mode, msmtp reads a mail from standard input and sendscit to an SMTP server for delivery.
+
+- Send a mail using the default account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp {{to@example.org}}`
+
+- Send a mail using a specific account configured in `~/.msmtprc`:
+
+`echo {{"Hello world"}} | msmtp --account={{example}} {{to@example.org}}`
+
+- Send a mail without a configured account:
+
+`echo {{"Hello world"}} | msmtp --host={{localhost}} --port={{999}} --from={{from@example.org}} {{to@example.org}}`
```

Comment:
Maybe I'm misunderstanding how this works. If you did `msmtp --host=smtp.gmail.com --from=josa42@gmail.com waldyrious@gmail.com`, would this command prompt you for the password of your gmail account? Otherwise, the email surely would arrive at the destination with different parameters (SMTP headers I suppose?).